src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_product-cards/assets/106397016/554dd5bf-22b7-4ac4-a2e0-3e4c8cf5e6ef)

Reply:
Thank you for the review @VenomMB 👌